File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        l = []
        for i in nums:
            count = 0
            for j in range(len(nums)):
                if i == nums[j]:
                    count += 1
            l.append(count)
        count = 0
        for i in l:
            if max(l) == i:
                count += 1
        return count

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
import java.util.*;

class Solution {
    public int maxFrequencyElements(int[] nums) {
        List<Integer> freqList = new ArrayList<>();
        for (int i = 0; i < nums.length; i++) {
            int count = 0;
            for (int j = 0; j < nums.length; j++) {
                if (nums[i] == nums[j]) {
                    count++;
                }
            }
            freqList.add(count);
        }

        int maxFreq = Collections.max(freqList);
        int count = 0;
        for (int freq : freqList) {
            if (freq == maxFreq) {
                count++;
            }
        }
        return count;
    }
}
